libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/AppLog.java

```diff
@@ -236,8 +239,19 @@ private static String getStringStackTrace(Throwable throwable) {
     public static ArrayList<String> toHtmlList(Context context) {
         ArrayList<String> items = new ArrayList<String>();
 
+        PackageManager packageManager = context.getPackageManager();
```

Comment:
Rather than duplicate this code in `toPlainText()`, how about making it a simple routine instead?

```
private static String getAppName(Context context) {
    PackageManager packageManager = context.getPackageManager();
    PackageInfo pkInfo = PackageUtils.getPackageInfo(context);

    ApplicationInfo applicationInfo = pkInfo != null ? pkInfo.applicationInfo : null;
    if (applicationInfo != null && packageManager.getApplicationLabel(applicationInfo) != null) {
        return packageManager.getApplicationLabel(applicationInfo).toString();
    } else {
        return "Unknown";
    }
}
```